sdk/go/common/resource/plugin/langruntime_plugin.go

```diff
@@ -420,32 +389,26 @@ func (h *langhost) GetProgramDependencies(info ProgInfo, transitiveDependencies
 		})
 	}
 
-	logging.V(7).Infof("%s success: #versions=%d", prefix, len(results))
+	logging.V(7).Infof("%s success: #versions=%d", info, len(results))
```

Comment:
This doesn't log the same as before (no langhost[].GetProgramDependencies text because you changed what prefix means)

Reply:
Fixed ✅